nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -325,9 +325,11 @@ def upload_folder(self, args, ctx: CommandContext):
 
         # sign folders and files
         api = ctx.get_api()
-        client_key_file_path = api.client_key
-        private_key = load_private_key_file(client_key_file_path)
-        sign_folders(full_path, private_key, api.client_cert)
+        if api.poc_key != "admin":
```

Comment:
Better to extract "admin" to a constant, since it could be changed in AdminAPI.

also extra line 341 need to be removed:
        api = ctx.get_api()

Reply:
Thanks.  Fixed in the update.